### DISCUSSION TITLE
Help please with ERROR    Error running git: F:\automatic\repositories\stable-diffusion-stability-ai

### DISCUSSION THREAD

**Discussion by cgidesign-de** [vladmandic/sdnext#1826] [Q&A]
Hi,
since recently I get a bunch of git errors. It happen with argument: _F:\automatic\webui.bat --upgrade_ added to the webui batch file.

INFO     Verifying repositories
ERROR    Error running git: F:\automatic\repositories\stable-diffusion-stability-ai / pull origin main
                         --autostash --rebase --force
ERROR    Error running git: F:\automatic\repositories\taming-transformers / pull origin master
                         --autostash --rebase --force
ERROR    Error running git: F:\automatic\repositories\CodeFormer / fetch
ERROR    Error running git: F:\automatic\repositories\CodeFormer / checkout 7a584fd
ERROR    Error running git: F:\automatic\repositories\BLIP / pull origin main --autostash --rebase
                         --force
INFO     Verifying submodules

Any idea how to solve this?

**Comment by vladmandic** [comment]
And installed also prints to check log file for details, what does it say?

Btw, DO NOT EDIT webui.bat.
Read FAQ what to edit if you want to add a flag.

**Comment by cgidesign-de** [comment]
**Update:**
Git error also happens without --upgrade flag

**"DO NOT EDIT webui.bat"**
Not done that - argument is set in desktop shortcut of webui.bat

**Log:**
(I pasted only one set of messages, as the others are the same).
New here compared to the cli window is "index file corrupt".
I guess the index file in "F:\automatic\.git" is meant here?

--------------
| sd | INFO | installer | Verifying repositories
| sd | DEBUG | installer | Submodule: F:\automatic\repositories\stable-diffusion-stability-ai / main
| sd | ERROR | installer | Error running git: F:\automatic\repositories\stable-diffusion-stability-ai / pull origin main --autostash --rebase --force
| sd | DEBUG | installer | Git output: error: bad signature 0x00000000
fatal: index file corrupt
--------------

**Reply by vladmandic** [reply]
you `.git` folder has corrupt entries. i would delete installation and clone it again.

**Comment by cgidesign-de** [comment]
Yep, cloning from scratch did the trick - thanks.
